frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -1338,7 +1341,8 @@ const IR::Statement* ProgramStructure::convertPrimitive(const IR::Primitive* pri
 
 const IR::P4Action*
 ProgramStructure::convertAction(const IR::ActionFunction* action, cstring newName,
-                                const IR::Meter* meterToAccess) {
+                                const IR::Meter* meterToAccess,
+                                cstring counterToAccess) {
```

Comment:
I believe P4_14 only supports a single direct meter and a single direct counter on a table (though it is not really specified).  This code only supports one direct meter and one direct counter per table.